cmd/up/up.go

```diff
@@ -540,5 +575,11 @@ func printDisplayContext(dev *model.Dev, divertURL string) {
 	if divertURL != "" {
 		log.Println(fmt.Sprintf("    %s       %s", log.BlueString("URL:"), divertURL))
 	}
+	if len(endpoints) > 0 {
+		log.Println(fmt.Sprintf("    %s %s", log.BlueString("Endpoints:"), endpoints[0]))
+		for i := 1; i < len(endpoints); i++ {
+			log.Println(fmt.Sprintf("               %s", endpoints[i]))
```

Comment:
Is there a more straight forward option with `log.Printf`?

Reply:
If so, possibly fix the other logs unless you'd rather just focus on this change.